src/vs/workbench/api/node/proxyResolver.ts

```diff
@@ -77,11 +77,20 @@ export function connectProxyResolver(
 }
 
 function createPatchedModules(params: ProxyAgentParams, resolveProxy: ReturnType<typeof createProxyResolver>) {
+
+	function proxyAssign(module: any, patch: any) {
```

Comment:
@jrieken Any reason for not adding a setter for this proxy? 

Due to this change, our Postman extension is failing in the latest Insider Release, as we need to re-assign `tls.createSecureContext` for https://github.com/postmanlabs/postman-request/blob/00acf088467643fafb5b5c5bb0100f05a6ec5cca/index.js#L185

https://github.com/postmanlabs/postman-app-support/issues/12923